src/stash/src/tests.rs

```diff
@@ -759,6 +561,210 @@ where
     stash
 }
 
+#[mz_ore::test(tokio::test)]
+#[cfg_attr(miri, ignore)] // unsupported operation: can't call foreign function `TLS_client_method` on OS `linux`
+async fn test_stash_fail_after_commit() {
+    mz_ore::test::init_logging_default("debug");
```

Comment:
Just a thought, I wonder if we should update the `mz_ore::test` macro to support specifying the logging level?